op-e2e/actions/helpers/l2_verifier.go

```diff
@@ -130,7 +130,7 @@ func NewL2Verifier(t Testing, log log.Logger, l1 derive.L1Fetcher,
 
 	var interopSys interop.SubSystem
 	if cfg.InteropTime != nil {
-		interopSys = managed.NewManagedMode(log, cfg, "127.0.0.1", 0, interopJWTSecret, l1, eng)
+		interopSys = managed.NewManagedMode(log, cfg, "127.0.0.1", 0, interopJWTSecret, l1, eng, nil)
```

Comment:
Why a `nil` func here?

Reply:
whoops, I should have left a `//TODO` on this while working. Will fix.

Reply:
`executor` appears to satisfy the Drainer interface, so I passed that.